docs/rancher/04-setup-rke2-cluster/README.md

```diff
@@ -0,0 +1,103 @@
+
+**Note**: For a HA setup, deploy an odd number of servers, at least three, for both `etcd` and `control plane` roles to ensure redundancy and avoid split-brain scenarios. This setup maintains control plane availability for managing cluster operations even if a node fails.
+
+## Step 1 - Accessing the Rancher Dashboard
+
+Begin by opening a web browser and navigating to `http://<rancher_url>`, substituting `<rancher_url>` with your specific Rancher URL. Log in using the credentials established during the Rancher installation.
+
+## Step 2 - Initiating Cluster Creation
+
+Within the Rancher dashboard, proceed to the **Cluster Management** tab and click **Create**.
+
+![Initiating cluster creation step 1](./assets/images/initiating_cluster_creation_1.png)
+
+Select the **Custom** option to commence the creation of a new cluster.
+
+![Choosing the custom option for new cluster](./assets/images/initiating_cluster_creation_2.png)
+
+Input the **Cluster Name** and press **Create** to move to the subsequent step.
+
+![Entering cluster name](./assets/images/initiating_cluster_creation_3.png)
+
+## Step 3 - Registering Nodes to the Cluster
+
+Navigate to your new cluster's configuration page.
+
+![Accessing new cluster configuration](./assets/images/registering_nodes_1.png)
+
+On this page, select the **Registration** tab.
+
+![Registration tab selection](./assets/images/registering_nodes_2.png)
+
+This section provides the command to run on the Ubuntu servers designated for inclusion in your Kubernetes cluster.
+
+![Command for node registration](./assets/images/registering_nodes_3.png)
+
+Open a terminal, connect to your server via SSH, and execute the command provided by Rancher.
+
+![Paste Command](./assets/images/registering_nodes_4.png)
+
+Revisit the Rancher interface and click on the **Machine** tab; your server will be listed and will require a few minutes to initialize.
+
+![Machine Tab](./assets/images/registering_nodes_5.png)
+
+## Step 4 - Retrieving the Cluster Kubeconfig
+
+To manage your cluster with kubectl from your local environment, you need to download the kubeconfig file from Rancher.
+
+Navigate to **Cluster Management** in Rancher.
+
+![Access Cluster Management](./assets/images/downloading_the_cluster_kubeconfig_file_1.png)
+
+Locate your cluster, click the `...` next to the **Explore** button, and choose **Download Kubeconfig**.
+
+![Download Kubeconfig](./assets/images/downloading_the_cluster_kubeconfig_file_2.png)
+
+Follow the guide on [Merging Configuration Files for Cluster Management](../02-setup-k3s/README.md#step-5---merging-configuration-files-for-cluster-management) to integrate this kubeconfig with any existing configurations.
+
+Alternatively, use the following commands to set up your kubeconfig for cluster management. Replace `<cluster_name>.yaml` with the filename of your downloaded kubeconfig:
```

Comment:
One step is missing here: when the user download kubeconfig to a path, for instance: Desktop. They need to move it to `~/.kube` before executing the following commands.

Reply:
done